docs/step_by_step_guide.md

```diff
@@ -0,0 +1,649 @@
+done
+```
+
+> [!TIP]
+> If you want to check a long simulation, you can use the `--tars` option to split the checking into more manageable pieces:
+> ```shell
+> # Starting at 00005a until the end
+> zstash check --tars=00005a-
+> # Starting from the beginning to 00005a (included)
+> zstash check --tars=-00005a
+> # Specific range
+> zstash check --tars=00005a-00005c
+> # Selected tar files
+> zstash check --tars=00003e,00004e,000059
+> # Mix and match
+> zstash check --tars=000030-00003e,00004e,00005a-
+> ```
+
+Then, do the following:
+```shell
+$ ssh dtn01.nersc.gov
+$ screen
+$ screen -ls # Output should say "Attached"
+$ cd /global/homes/<first letter>/<username>
+$ ./batch_zstash_check.bash
+# Control A D to exit screen
+# DO NOT CONTROL X / CONTROL C (as for emacs). This will terminate the task running in screen!!!
+$ screen -ls # Output should say "Detached"
+$ hostname
+# If you log in on another login node, 
+# then you will need to ssh to this one to get back to the screen session.
+# Wait for the script to finish
+# (On Chrysalis, for 165 years of data, this takes ~5 hours)
+$ screen -r # Return to screen
+# Check that output ends with `INFO: No failures detected when checking the files.`
+# as well as listing real`, `user`, `sys` time information
+$ exit # Terminate screen
+$ screen -ls # The screen should no longer be listed
+$ exit # exit data transfer node
+$ cd /global/cfs/cdirs/e3sm/<username>/E3SMv2/<case_name>/zstash
+$ tail zstash_check_<stamp>.log
+# Output should match the output from the screen (without the time information)
+```
+
+## 4. Document
+On a NERSC machine (Perlmutter):
+```shell
+$ hsi
+$ ls /home/<first letter>/<username>/E3SMv2
+# Check that the simulation case is now listed in this directory
+$ ls /home/<first letter>/<username>/E3SMv2/<case_name>
+# Should match the number of files in the other machine's `<simulations_dir>/<case_name>/zstash`
+$ exit
+$ cd /global/cfs/cdirs/e3sm/<username>/E3SMv2/<case_name>/zstash
+$ ls
+# `index.db` and `zstash_check` log should be the only items listed
+# https://www2.cisl.ucar.edu/resources/storage-and-file-systems/hpss/managing-files-hsi
+# cput will not transfer a file if it exists.
+$ hsi "cd /home/<first letter>/<username>/E3SMv2/<case_name>; cput -R <zstash_check log>"
+$ hsi
+$ ls /home/<first letter>/<username>/E3SMv2/<case_name>
+# tar files, `index.db`, `zstash_create` log, and `zstash_check` log should be present
+$ exit
+```
+Update the simulation Confluence page with information regarding this simulation (For Water Cycle’s v2 work, that page is [V2 Simulation Planning](https://acme-climate.atlassian.net/wiki/spaces/ED/pages/2766340117)). In the `zstash archive` column, specify: 
+- `/home/<first letter>/<username>/E3SMv2/<case_name>`
+- `zstash_create_<stamp>.log`
+- `zstash_check_<stamp>.log`
+
+<!-- TODO: Is there a v3 planning page to link? -->
```

Comment:
Is there a corresponding v3 simulation planning page with information about all the simulations?